kuiper-game/MainRegistry.cs

```diff
@@ -10,6 +12,9 @@ public class MainRegistry : ServiceRegistry
         public MainRegistry()
         {
             this.AddHostedService<MainLoopWorker>();
+            this.AddSingleton<ISolarSystemRepository>(x => 
+                ActivatorUtilities.CreateInstance<JsonFileSolarSystemRepository>(x, new FileInfo("Data\\Sol.solarsystem")));
```

Comment:
Why use reflection to create the instance?

Reply:
I couldn't figure out how to get the filename in.

Reply:
(x => new JsonFileSolarSystemRepository(new FileInfo))?

Reply:
> (x => new JsonFileSolarSystemRepository(new FileInfo))?.

Hah, I swear I tried that. Or maybe it was just something a little bit like that :)
Fixed now.